src/App/docutils/ComponentPreview/index.js

```diff
@@ -153,10 +153,163 @@ const ComponentPreview = ({ children, language, removeOuterTag, hideValue, remov
         </div>
     );
 
+    const { tabs } = window.dg;
+
+    class ShowCasePanelAdvanced extends Component {
```

Comment:
Complicated, but effective class. As you said on slack, we can later go over how to make the developer experience, especially regarding props, as easy as possible.